docs/src/css/quasar.variables.sass

```diff
@@ -9,6 +9,8 @@
 // to match your app's branding.
 // Tip: Use the "Theme Builder" on Quasar's documentation website.
 
+@import 'landingpage/variables.sass'
```

Comment:
For multi-word names let's keep a dash-cased convention as it's the standard in CSS realm, so this should be `landing-page` everywhere